external/wakaama/core/er-coap-13/er-coap-13.c

```diff
@@ -913,6 +913,9 @@ coap_status_t coap_parse_message(void *packet, coap_protocol_t protocol, uint8_t
 		PRINTF("OPTION %u (delta %u, len %u): ", option_number, option_delta, option_length);
 		SET_OPTION(coap_pkt, option_number);
 
+		if (current_option + option_length > (uint8_t *)data + data_len) {
```

Comment:
@line no 883,  similar check is done for  "<" less than data+data_len  ,  If am not wrong, This check should be done as ">=" instead of just ">" 
Kindly double check.